ledger/participant-state/kvutils/tools/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
Enables mocking final classes.

Reply:
Urgh. Can't see a better way around this though.

Perhaps a future implementation might add a trait and make the state updates injectable.